docs/pages/docs/guide/built-ins.mdx

```diff
@@ -18,24 +19,29 @@ your content:
     title="Callout"
     href="/docs/guide/built-ins/callout"
   />
-  <Cards.Card
-    icon={<CardsIcon />}
-    title="Tabs"
-    href="/docs/guide/built-ins/tabs"
-  />
   <Cards.Card
     icon={<IdCardIcon />}
     title="Cards"
     href="/docs/guide/built-ins/cards"
   />
+  <Cards.Card
+    icon={<FolderTreeIcon className="size-5" />}
+    title="FileTree"
+    href="/docs/guide/built-ins/filetree"
+  />
   <Cards.Card
     icon={<OneIcon />}
     title="Steps"
     href="/docs/guide/built-ins/steps"
   />
   <Cards.Card
-    icon={<FolderTreeIcon className="size-5" />}
-    title="FileTree"
-    href="/docs/guide/built-ins/filetree"
+    icon={<CardsIcon />}
+    title="Tabs"
+    href="/docs/guide/built-ins/tabs"
+  />
+  <Cards.Card
+    icon={<TableIcon />}
+    title="Table"
+    href="/docs/guide/built-ins/table"
```

Comment:
let's move table first and tabs after, to be consistent with _meta file (I'll push to your PR)

Reply:
That makes sense :) thank you!